lib/assessment.rb

```diff
@@ -20,20 +26,30 @@
 # Name and location should be writeable, but age should be read-only.
 
 ##
-# your code here
+class Person
+  attr_writer :name, :location
```

Comment:
Actually, this should be `attr_accessor`, not `attr_writer`. `attr_writer` will only give you write access to those variables, not read access.